### PR TITLE
[Google Blockly] use thrasos as default renderer

### PR DESCRIPTION
**Pull request by mikeharv**
Updates our Google Blockly labs to use the thrasos renderer by default, unless overridden with an experiment.

Unblocks:
* https://github.com/code-dot-org/code-dot-org/pull/51066

## About the Thrasos renderer
Thrasos (Greek: "bold") is the name of Blockly's modern replacement for the Geras (Greek: "old") renderer that we use today in most labs. It has a flatter, more modern look, and improved vertical alignment of text on taller blocks. Geras blocks have a 3d look that is accomplished through an extra SVG "highlight". C-shaped blocks also have a more symmetrical shape.

**_Geras:_**
imageimage

**_Thrasos:_**
imageimage

Thrasos is the current default renderer for Music Lab and we would like to start using it other labs as well. In addition to the desired modernization and consistency this would bring, it would also simplify some complexity that is currently blocking Sprite Lab migration work (https://github.com/code-dot-org/code-dot-org/pull/51066).

Today, the Blockly team explicitly recommends this renderer for new projects although they still support Geras for projects that already use it as well as the Zelos renderer, which was explicitly designed to mimic Scratch.

This change updates the default renderer for all Google Blockly labs to consistently use Thrasos, including main workspaces created through the standard inject and readOnlyBlockSpaces, which are used in places like instructions. To test our confidence with this change, I made sure all blocks rendered as expected in the Dance and Poetry block pools, and compared various Google Blockly lab projects.

### Side-by-side comparisons

| Geras | Thrasos|
|-|-|
|flappy geras|flappy thrasos|
|poemart geras|poemart thrasos|
|highcontrast geras|highcontrast thrasos|

## Testing story

No functional changes. It's possible that this could contribute to eyes diffs, so we will communicate with the DOTD when this is merged.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by bencodeorg — APPROVED** [review]

**Review by breville — APPROVED** [review]

File: apps/src/StudioApp.js [hunk 1]
```diff
@@ -1212,11 +1212,11 @@ StudioApp.prototype.inject = function (div, options) {
...
   };
 
-  // Allows Google Blockly labs to use the Zelos or Thrasos renderer instead of the default.
+  // Allows Google Blockly labs to use the Zelos or legacy Geras renderer instead of the default Thrasos.
   if (experiments.isEnabled('zelos')) {
     options.renderer = 'cdo_renderer_zelos';
-  } else if (experiments.isEnabled('thrasos')) {
-    options.renderer = 'cdo_renderer_thrasos';
+  } else if (experiments.isEnabled('geras')) {
+    options.renderer = 'cdo_renderer';
```
Comment (fisher-alice) [inline apps/src/StudioApp.js:1219]: What do you think of renaming `cdo_renderer` to `cdo_renderer_geras` and also renaming the class `CdoRenderer` to `CdoRendererGeras`?

**Review by fisher-alice — APPROVED** [review]
This is exciting! And it'll be nice to have consistency across Google Blockly labs. 
Just a small renaming suggestion.

File: apps/src/StudioApp.js (hunk 1 above)
Comment (mikeharv) [reply apps/src/StudioApp.js:1219]: That sounds good to me. It looks like you’ve already prepped that change in your branch so I’ll merge this one as is.

Comment (fisher-alice) [reply apps/src/StudioApp.js:1219]: Sounds good - thanks! For documentation: [prepped branch](https://github.com/code-dot-org/code-dot-org/tree/alice/sprite-lab-socket-shapes)

**Pull request merged**